product-service/src/utils/responseBuilder.ts

```diff
@@ -0,0 +1,33 @@
+import { APIGatewayProxyResult } from 'aws-lambda';
+
+const corsHeaders = {
+  'Access-Control-Allow-Methods': '*',
+  'Access-Control-Allow-Headers': '*',
+  'Access-Control-Allow-Origin': '*'
+};
+
+const successResponse = (body: Object, statusCode: number = 200): APIGatewayProxyResult => {
```

Comment:
i believe, that 200 code as default is not a very good idea

Reply:
Why not? It's a successful response.